src/App.js

```diff
@@ -1,24 +1,13 @@
-import logo from './logo.svg';
+import * as React from 'react';
+import Container from '@mui/material/Container';
+import Recipes from './components/Recipes';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Container>
+      <Recipes></Recipes>
```

Comment:
When there are no child components, we can write components as `<ComponentName />` for short:
```suggestion
      <Recipes />
```

Reply:
Perfect, fixing 👌